app/services/users.js

```diff
@@ -0,0 +1,27 @@
+const { User } = require('../models');
+const errors = require('../errors');
+
+const getUserByEmail = email => {
+  try {
+    return User.findAll({
+      where: {
+        email
+      }
+    });
+  } catch {
+    throw errors.databaseError('Error querying users by email');
+  }
+};
+
+const createUser = user => {
```

Comment:
I would think it would be a little cleaner this way

```
exports.createUser = user =>
  User.create(user).catch(err => {
    logger.error(`Failed to create user. Error: ${JSON.stringify(err)}`);
    throw errors.databaseError('An error was occurred when try to create a user. Database error');
  });
```

Reply:
try/catch block changed to catch function